src/Certify.Core/Models/ManagedItem.cs

```diff
@@ -155,6 +155,12 @@ public ManagedSite()
             this.RequestConfig.EnableFailureNotifications = true;
 
             this.RequestConfig.PropertyChanged += RequestConfig_PropertyChanged;
+            this.PropertyChanged += ManagedSite_PropertyChanged;
+        }
+
+        private void ManagedSite_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == "Comments") IsChanged = true;

```

Comment:
This change notification should have happened automatically - we're using Fody PropertyChanged to wire up the model as a bindable and enabled for INotifyPropertyChanged

Reply:
I've updated that to fire the change on type rather than focus change.

Reply:
Btw, I'm using CodeMaid to auto format etc so that why I randomly seem to be update the formatting of comments!

Reply:
Thanks! I was wondering about those comment whitespaces. Have you seen [EditorConfig](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.EditorConfig)? If you create the .editorconfig rule file and add it to the solution it'd help help everyone committing keep to common code conventions.